GDJS/Runtime/events-tools/commontools.js

```diff
@@ -106,16 +106,38 @@ gdjs.evtTools.common.logicalNegation = function(bool) {
   return !bool;
 };
 
+/**
+ * Limit an value in a range
+ * @param {number} x Value.
+ * @param {number} min Minimum value.
+ * @param {number} max Maximum value.
+ * @returns {number} The new value.
+ */
 gdjs.evtTools.common.clamp = function(x, min, max) {
   return Math.min(Math.max(x, min), max);
 };
 
+/**
+ * Hyperbolic arccosine
+ * @memberof gdjs.evtTools
+ * @class common
```

Comment:
The class tag is used to say a class is defined here, not that it is a member. Also, common is a namespace not a class.